### DISCUSSION TITLE
where can I run it online 24/7

### DISCUSSION THREAD

**Discussion by bytreakz** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#587] [Q&A] [answered]
Hey, is there a free (cheap or with a student license) hosting service where I can run the script 24/7 or 24h and start it everyday from my phone?

**Comment by ColinShark** [comment, marked as answer]
I use [Uberspace.de](https://uberspace.de/en). They're shared hosting, but PWYW with 1€ minimum. You can use a junk email if need be. Full linux user-account on a server with shared resources (ie other users on the same server).

**Reply by bytreakz** [reply]
do you know if it is safe to run it on replit.com because of the password in the run.py (have the Hacker plan) and put my repl on private?

**Reply by ColinShark** [reply]
I've never used that. No experience whatsoever with that. I ran my script locally first to get the cookie file (`./cookies/username.pkl`), which I then put on my uberspace. Runs perfectly fine 24/7. I have channels with over 350k points from *just* the script itself.

**Reply by bytreakz** [reply]
Ok, guess that I'll switch to uberspace in December, can I get in touch with you if I need help with it?

**Reply by ColinShark** [reply]
Uberspace is *very* self-explanatory. They also have a good [manual](https://manual.uberspace.de), as well as a [lab](https://lab.uberspace.de) with tutorials for different software.

**Reply by taurynamonsterxdd** [reply]
@ColinShark can you help me how can I do it? Pls

**Reply by taurynamonsterxdd** [reply]
who can help me to run my script on uberspace plssssssssssssss!

**Comment by Rezanans-wow** [comment]
I use pterohost Discord bot hosting, i asked they allow host it. Cost lower 1.5 EUR

**Reply by Guliveer** [reply]
Oracle free tier is a way to go! (For me at least)

**Reply by bytreakz** [reply]
> Oracle free tier is a way to go! (For me at least)

I'll try it when my repl expires 👍🏼 is it running 24/7 in Oracle?

**Reply by Guliveer** [reply]
> > Oracle free tier is a way to go! (For me at least)
> 
> I'll try it when my repl expires 👍🏼 is it running 24/7 in Oracle?

Yes, it's running 24/7. However you have to set it as a service/daemon using systemd. Then it works perfectly!
(Currently twitch drops and bonus points are not working, but it's for another topic. Not related to daemon tho)

Lmk if you need some template file for that 😉

**Reply by taurynamonsterxdd** [reply]
who can help me to run my script on uberspace plssssssssssssss!

**Reply by taurynamonsterxdd** [reply]
@Guliveer

**Reply by Guliveer** [reply]
> @Guliveer 
> 
> 

I have no idea how Uberspace works, therefore I'm unable to help you

**Reply by taurynamonsterxdd** [reply]
what vps do you usue

**Reply by Guliveer** [reply]
> what vps do you usue
> 
> 
> 
> 

Read the damn thread

**Comment by roflankaban** [comment]
Can it be deployed on Heroku?

**Reply by Guliveer** [reply]
> Can it be deployed on Heroku?

Of course it can. But now Heroku is removing free tier.

**Reply by taurynamonsterxdd** [reply]
who can help me to run my script on uberspace plssssssssssssss!

**Comment by taurynamonsterxdd** [comment]
who can help me to run my script on uberspace plssssssssssssss!

**Reply by Yishaqel** [reply]
I guess Google can help you
